modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -323,16 +342,17 @@ imageContentSources: <14>
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
 ====
-<5> The machine CIDR must contain the subnets for the compute machines and control plane machines.
-<6> The CIDR must contain the subnets defined in `platform.ibmcloud.controlPlaneSubnets` and `platform.ibmcloud.computeSubnets`.
-<7> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<8> The name of an existing resource group. The existing VPC and subnets should be in this resource group. The cluster is deployed to this resource group.
-<9> Specify the name of an existing VPC.
-<10> Specify the name of the existing VPC subnet. The subnets must belong to the VPC that you specified. Specify a subnet for each availability zone in the region.
-<11> For `<local_registry>`, specify the registry domain name, and optionally the port, that your mirror registry uses to serve content. For example, registry.example.com or registry.example.com:5000. For `<credentials>`, specify the base64-encoded user name and password for your mirror registry.
-<12> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
-<13> Provide the contents of the certificate file that you used for your mirror registry.
-<14> Provide the `imageContentSources` section from the output of the command to mirror the repository.
+<5> The SMTLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are from 1 to 8.
```

Comment:
You cannot set it to any value between 1 and 8.  For e.g. 3, 5, 6, 7 etc.  Only Powers of 2 are allowed, 1, 2, 4, 8.

Reply:
> You cannot set it to any value between 1 and 8. For e.g. 3, 5, 6, 7 etc. Only Powers of 2 are allowed, 1, 2, 4, 8.

As per @mjturek  all values from 1 to 8 are supported on Power.
QE tested and verified this with smtLevel set to 5 and 7. Cluster is stable in both cases.

With smtLevel set to 7, lscpu on the node shows
```
Architecture:            ppc64le
  Byte Order:            Little Endian
CPU(s):                  8
  On-line CPU(s) list:   0-6
  Off-line CPU(s) list:  7
Model name:              POWER9 (architected), altivec supported
  Model:                 1.3 (pvr 004e 2103)
  Thread(s) per core:    7
  Core(s) per socket:    1
  Socket(s):             1
  Physical sockets:      16
  Physical chips:        1
  Physical cores/chip:   10
```


With smtLevel set to 5, lscpu on the nodes show this,
```
Architecture:            ppc64le
Byte Order:            Little Endian
CPU(s):                  8
  On-line CPU(s) list:   0-4
  Off-line CPU(s) list:  5-7
Model name:              POWER9 (architected), altivec supported
  Model:                 1.3 (pvr 004e 2103)
  Thread(s) per core:    5
  Core(s) per socket:    1
  Socket(s):             1
  Physical sockets:      16
  Physical chips:        1
  Physical cores/chip:   10
```

fyi: @manojnkumar @alishaIBM